source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,361 @@
+In addition, we will want to simplify the URL for the user so that they can access the information for a specific rental by browsing to `localhost:4200/rentals/slug-for-rental`.
+To do this, we modify our router.
+
+```app/router.js
+Router.map(function() {
+  this.route('about');
+  this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:slug' });
+  });
+});
+```
+
+We have modified the `path` for the `show` route.
+It no longer defaults to `/rentals/show` but instead `/rentals/:slug`.
+This means we can now access the `slug` on the route itself.
+
+# Querying by Slug
+
+Next, we will want to edit our `show` route to return the specific rental we want.
+
+```app/routes/rentals/show.js
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { slug: params.slug });
+  }
+});
+```
+
+Since we added `:slug` to the `show` path in our router, we can now access `slug` through the `params` in our `model` hook.
+When we call `this.store.queryRecord('rental', { slug: params.slug })`, Ember Data will make a GET request to `/rentals?slug=our-slug`.
+You can read more about Ember Data in the [Models section](../../models/).
+
+This means that we can access a specific rental as `model` on our template.
+Optionally, we can add a `setupController` call to rename our `model` and make our template more descriptive.
+This isn't necessary, but allows us rename `model` to `rental`, making the template easier to read.
+
+```app/routes/rentals/show.js
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { slug: params.slug });
+  },
+
+  setupController(controller, model) {
+    Ember.set(controller, 'rental', model);
+  }
+});
+```
+
+Since we are editing the default behavior to rename our model, we should write an appropriate test to ensure we get the results we want.
+Let's open the unit test that Ember generated for us when we created our `show` route.
+
+```tests/unit/routes/rentals/show.js
+moduleFor('route:rentals/show', 'Unit | Route | rentals/show', {});
+
+test('`setupController` sets rental from our model', function(assert) {
+  assert.expect(1);
+  const route = this.subject();
+
+  const model = {
+    title: "Grand Old Mansion",
+    owner: "Veruca Salt",
+    city: "San Francisco"
+  };
+
+  const controller = {};
+
+  route.setupController(controller, model);
+
+  const expectedController = {
+    rental: model
+  };
+
+  assert.deepEqual(controller, expectedController, 'rental is added');
+});
+
+```
+
+Here, we ensure that if our model returns an object, that it is renamed to `rental` on our controller.
+
+# Adding the Retnal To Our Template
+
+Next, we can update the template for our show route (`app/templates/rentals/show.hbs`) and list the information for our rental.
+
+```app/templates/rentals/show.hbs
+{{rental-listing rental=rental}}
```

Comment:
May need to revise later based on https://github.com/emberjs/super-rentals/pull/24#discussion_r64213804